mission_control/test/mission_interface.py

```diff
@@ -108,10 +108,11 @@ def load_mission(self, mission_name):
         self.mission_load_state_flag = False
         mission_to_load = LoadMission()
         mission_to_load.mission_file_full_path = self.full_path
-        self.simulated_mission_control_load_mission_pub.publish(mission_to_load)
+        self.simulated_mission_control_load_mission_pub.publish(
+            mission_to_load)
 
         def load_mission_state():
-            return self.mission_load_state == ReportLoadMissionState.SUCCESS
+            return self.mission_load_state == load_status
```

Comment:
@patapufate nit^N: if `MissionInterface.load_mission()` would wait and return the load state, the test would read better e.g. `self.assertEq(load_mission(...), SUCCESS)`

Reply:
Changed